synapse/lib/stormtypes.py

```diff
@@ -8017,6 +8021,14 @@ async def _methWipeLayer(self):
         view = self.runt.snap.core.getView(viewiden)
         await view.wipeLayer(useriden=useriden)
 
+    async def _methSwapLayer(self):
+
+        iden = self.valu.get('iden')
```

Comment:
just realized i need an extra check here for the view being a fork. update inbound.